benchmarks/cluster3nodes/configuration/parameters

```diff
@@ -6,7 +6,7 @@ numtx=1000
 targetnodes="0 1 2"
```

Comment:
I think this file should not be changed by the PR

Reply:
Done.